docs/system-admin-guide/integrations/nextcloud/oidc-sso/README.md

```diff
@@ -74,51 +72,46 @@ to your OpenProject and Nextcloud instances via Keycloak, you should already hav
 
-Next, we need to apply the policies, so that they actually allow the token exchange for given clients. Navigate to the `openproject` client, go to the Tab "Permissions", enable the "Permissions enabled" checkbox and select the permission with the name `token-exchange`. You should end up on a form similar to this one:
+Navigate to "Clients", choose `openproject` and then go to the Tab called "Client Scopes". Press the button to "Add client scope" and select the `add-nextcloud-audience` scope, make sure it is set to "Optional", so that the scope is only included when it was requested. Afterwards go to the `nextcloud` client and add the scope `api_v3` there the same way.
 
-![Empty view of the permission configuration dialog](keycloak_exchange_permission.png)
+Both clients are now able to request the given scopes during token exchange.
 
-In the field "Policies" you can now add the `allowed-to-exchange-openproject-token` we created earlier. This means that clients matching that policy will be allowed to exchange an access token that includes the `openproject` audience.
+##### Configuring OpenProject
 
-Afterwards repeat the previous steps to add the `allowed-to-exchange-nextcloud-token` policy to the `nextcloud` client.
+In OpenProject we need to configure the ID of the Nextcloud client (in our example: `nextcloud`) as the "Storage Audience".
+This will make sure that OpenProject will attempt to exchange a token for that audience before making requests to Nextcloud.
 
-Lastly, we need to allow Nextcloud to obtain the previously created `api_v3` scope when requesting a token. Navigate to "Clients", choose
-`nextcloud` and then go to the Tab called "Client Scopes". Press the button to "Add client scope" and select the `api_v3` scope, make sure
-it is set to "Optional", so that the scope is only included when it was requested. Afterwards to to the `openproject` client and add the
-scope there the same way. This is required so that exchanged tokens can carry the scope as well.
+We also need to request the previously created `add-nextcloud-audience` scope during Token Exchange. To do that configure the "Storage Scope" to `add-nextcloud-audience`:
 
-##### Configuring OpenProject
+![OpenProject is configured to manually exchange tokens for the audience called nextcloud requesting a scope called add-nextcloud-audience](openproject_manual_storage_audience.png)
 
-In OpenProject it is enough to configure the ID of the Nextcloud client (in our example: `nextcloud`) as the "Storage Audience". This will make sure that OpenProject will attempt to exchange a token for that audience before making requests to Nextcloud:
-
-![OpenProject is configured to manually exchange tokens for the audience called nextcloud](openproject_manual_storage_audience.png)
+(TODO: update image)
```

Comment:
Note: The image can only be updated well after we merged the dependent feature. This ensures that it will look this way in merged state as well and allows us to take the screenshot on Edge, like the previous one.

Reply:
I am not going to merge this PR before the image is exchanged. So that we don't forget to exchange it xD

Reply:
image has been exchanged